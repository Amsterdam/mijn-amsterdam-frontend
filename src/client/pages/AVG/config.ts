import { AVGRequestFrontend } from '../../../server/services/avg/types';
import { dateSort } from '../../../universal/helpers/date';
import { LinkProps } from '../../../universal/types';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';

export const displayPropsAanvragen: DisplayProps<
  WithDetailLinkComponent<AVGRequestFrontend>
> = {
  detailLinkComponent: 'Nummer',
  ontvangstDatumFormatted: 'Ontvangen op',
  themas: 'Onderwerp(en)',
};

export const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    title: 'Lopende aanvragen',
    filter: (avgVerzoek: AVGRequestFrontend) => !avgVerzoek.datumAfhandeling,
    ...tableConfigBase,
  },
  [listPageParamKind.completed]: {
    title: 'Afgehandelde aanvragen',
    filter: (avgVerzoek: AVGRequestFrontend) => avgVerzoek.datumAfhandeling,
    ...tableConfigBase,
  },
};

export const linkListItems: LinkProps[] = [
  {
    to: 'https://www.amsterdam.nl/privacy/loket/',
    title: 'Loket persoonsgegevens gemeente Amsterdam',
  },
];
