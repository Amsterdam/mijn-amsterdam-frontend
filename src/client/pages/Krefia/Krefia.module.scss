.SectionBorderTop {
  border-top: 0.1rem $color-neutral-grey2 solid;
  padding-top: 2rem;
}

.HulpTable {
  tr td:first-child {
    width: 70%;
  }
}
