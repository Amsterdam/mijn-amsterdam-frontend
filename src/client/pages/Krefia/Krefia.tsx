import { Paragraph } from '@amsterdam/design-system-react';

import { useKrefiaThemaData } from './useKrefiaThemaData.hook';
import type { KrefiaDeepLink } from '../../../server/services';
import { entries } from '../../../universal/helpers/utils';
import { PageContentCell } from '../../components/Page/Page';
import ThemaPagina from '../ThemaPagina/ThemaPagina';
import ThemaPaginaTable from '../ThemaPagina/ThemaPaginaTable';

export function KrefiaThemaPagina() {
  const {
    title,
    deepLinks,
    hasFIBU,
    hasKrefia,
    hasKredietbank,
    tableConfig,
    linkListItems,
    isError,
    isLoading,
  } = useKrefiaThemaData();

  const pageContentTop = (
    <PageContentCell>
      {((hasFIBU && hasKredietbank) || !hasKrefia) && (
        <Paragraph>
          Een online plek waar u alle informatie over uw geldzaken kunt vinden
          als klant van Budgetbeheer (FIBU) en/of Kredietbank Amsterdam.
        </Paragraph>
      )}
      {hasKredietbank && !hasFIBU && (
        <Paragraph>
          Een online plek waar u alle informatie over uw geldzaken kunt vinden
          als klant van Kredietbank Amsterdam.
        </Paragraph>
      )}
      {!hasKredietbank && hasFIBU && (
        <Paragraph>
          Een online plek waar u alle informatie over uw geldzaken kunt vinden
          als klant van Budgetbeheer (FIBU).
        </Paragraph>
      )}
    </PageContentCell>
  );

  const krefiaTables = entries(tableConfig).map(
    ([kind, { title, displayProps, filter }]) => {
      return (
        <ThemaPaginaTable<KrefiaDeepLink>
          key={kind}
          title={title}
          zaken={deepLinks.filter(filter)}
          displayProps={displayProps}
        />
      );
    }
  );

  return (
    <ThemaPagina
      title={title}
      isError={isError}
      isLoading={isLoading}
      pageContentTop={pageContentTop}
      pageContentMain={krefiaTables}
      linkListItems={linkListItems}
    />
  );
}

// export default function Krefia() {
//   const { KREFIA } = useAppStateGetter();
//   const deepLinks = useDeepLinks(KREFIA.content?.deepLinks);
//   const isKredietbank = !!deepLinks?.schuldhulp || !!deepLinks?.lening;
//   const isFIBU = !!deepLinks?.budgetbeheer;
//   let showText = false;

//   if (!isKredietbank && !isFIBU) {
//     showText = true;
//   }
//   return (
//     <OverviewPageV2>
//       <PageContentV2>
//         <PageHeadingV2 backLink={AppRoutes.HOME}>
//           {ThemaTitles.KREFIA}
//         </PageHeadingV2>
//         {isLoading(KREFIA) && (
//           <PageContentCell>
//             <LoadingContent />
//           </PageContentCell>
//         )}

//         {isError(KREFIA) && (
//           <PageContentCell>
//             <ErrorAlert>

//             </ErrorAlert>
//           </PageContentCell>
//         )}
//       </PageContentV2>
//       {deepLinks?.schuldhulp && (
//         <SectionCollapsible
//           id="SectionCollapsible-krefia-schuldregeling"
//           title="Schuldregeling"
//           startCollapsed={false}
//           className={styles.SectionBorderTop}
//           isLoading={isLoading(KREFIA)}
//         >
//           <Table
//             className={styles.HulpTable}
//             displayProps={DISPLAY_PROPS}
//             items={deepLinks.schuldhulp}
//           />
//         </SectionCollapsible>
//       )}
//       {deepLinks?.lening && (
//         <SectionCollapsible
//           id="SectionCollapsible-krefia-leningen"
//           title="Leningen"
//           startCollapsed={!!deepLinks?.schuldhulp?.length}
//           isLoading={isLoading(KREFIA)}
//         >
//           <Table
//             className={styles.HulpTable}
//             displayProps={DISPLAY_PROPS}
//             items={deepLinks.lening}
//           />
//         </SectionCollapsible>
//       )}
//       {deepLinks?.budgetbeheer && (
//         <SectionCollapsible
//           id="SectionCollapsible-krefia-budgetbeheer"
//           title="Budgetbeheer"
//           startCollapsed={
//             !!deepLinks?.schuldhulp?.length || !!deepLinks?.lening?.length
//           }
//           isLoading={isLoading(KREFIA)}
//         >
//           <Table
//             className={styles.HulpTable}
//             displayProps={DISPLAY_PROPS}
//             items={deepLinks.budgetbeheer}
//           />
//         </SectionCollapsible>
//       )}
//     </OverviewPageV2>
//   );
// }
