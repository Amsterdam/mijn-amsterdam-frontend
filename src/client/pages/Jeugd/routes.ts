import { JeugdDetail } from './JeugdDetail';
import { JeugdThemaPagina } from './JeugdThema';
import { FeatureToggle } from '../../../universal/config/feature-toggles';

const baseRoute = '/jeugd';

export const JeugdRoutes = [
  {
    route: baseRoute,
    Component: JeugdThemaPagina,
    isActive: FeatureToggle.zorgnedLeerlingenvervoerActive,
  },
  {
    route: `${baseRoute}/voorziening/:id`,
    Component: JeugdDetail,
    isActive: FeatureToggle.zorgnedLeerlingenvervoerActive,
  },
];
