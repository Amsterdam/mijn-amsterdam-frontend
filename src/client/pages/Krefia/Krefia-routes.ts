import { KrefiaThemaPagina } from './Krefia';
import { FeatureToggle } from '../../../universal/config/feature-toggles';

export const KrefiaRoutes = [
  {
    route: '/kredietbank-fibu',
    Component: KrefiaThemaPagina,
    isActive: FeatureToggle.krefiaActive,
  },
];
