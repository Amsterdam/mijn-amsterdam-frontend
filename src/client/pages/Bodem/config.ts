import { generatePath } from 'react-router-dom';

import { LoodMetingFrontend } from '../../../server/services/bodem/types';
import { AppRoutes } from '../../../universal/config/routes';
import { dateSort } from '../../../universal/helpers/date';
import { LinkProps } from '../../../universal/types';

const displayPropsAanvragen = {
  detailLinkComponent: 'Adres',
  datumAanvraagFormatted: 'Aangevraagd',
  status: 'Status',
};

const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    ...tableConfigBase,
    title: 'Lopende aanvragen',
    filter: (bodemAanvraag: LoodMetingFrontend) => !bodemAanvraag.processed,
    listPageRoute: generatePath(AppRoutes['BODEM/LIST'], {
      kind: listPageParamKind.inProgress,
    }),
  },
  [listPageParamKind.completed]: {
    ...tableConfigBase,
    title: 'Afgehandelde aanvragen',
    filter: (bodemAanvraag: LoodMetingFrontend) => bodemAanvraag.processed,
    listPageRoute: generatePath(AppRoutes['BODEM/LIST'], {
      kind: listPageParamKind.completed,
    }),
  },
} as const;

export const linkListItems: LinkProps[] = [
  {
    title: 'Meer informatie over lood in de bodem.',
    to: 'https://www.amsterdam.nl/wonen-leefomgeving/bodem/lood-grond/',
  },
] as const;
