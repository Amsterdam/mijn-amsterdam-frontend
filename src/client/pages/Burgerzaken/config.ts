import { generatePath } from 'react-router-dom';

import { AppRoutes } from '../../../universal/config/routes';
import { dateSort } from '../../../universal/helpers/date';
import { IdentiteitsbewijsFrontend, LinkProps } from '../../../universal/types';
import { withOmitDisplayPropsForSmallScreens } from '../../components/Table/helpers';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';

const displayPropsBase: DisplayProps<
  WithDetailLinkComponent<IdentiteitsbewijsFrontend>
> = {
  detailLinkComponent: 'Type',
  datumAfloopFormatted: 'Geldig tot',
};

const listPageParamKind = {
  identiteitsbewijzen: 'identiteitsbewijzen',
};

const displayProps = withOmitDisplayPropsForSmallScreens(displayPropsBase, [
  'datumAfloopFormatted',
]);

export const tableConfig = {
  [listPageParamKind.identiteitsbewijzen]: {
    title: 'Mijn reisdocumenten',
    displayProps,
    sort: dateSort('datumAfloop', 'desc'),
    listPageRoute: generatePath(AppRoutes['BURGERZAKEN/LIST'], {
      kind: listPageParamKind.identiteitsbewijzen,
    }),
  },
} as const;

export const linkListItems: LinkProps[] = [
  {
    to: 'https://www.amsterdam.nl/burgerzaken',
    title: 'Overzicht en aanvragen bij burgerzaken',
  },
] as const;

export const backLinkDetailPage = AppRoutes.BURGERZAKEN;
