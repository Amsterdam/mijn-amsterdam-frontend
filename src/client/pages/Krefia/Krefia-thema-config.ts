import { KrefiaDeepLink } from '../../../server/services';
import { AppRoutes } from '../../../universal/config/routes';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';

const displayPropsDeeplink: DisplayProps<
  WithDetailLinkComponent<KrefiaDeepLink>
> = {
  status: 'Status',
  detailLinkComponent: 'Bekijk op Krefia',
};

export const routes = {
  themaPage: AppRoutes.KREFIA,
} as const;

export const krefiaTableConfig = {
  schuldhulp: {
    title: 'Schuldregeling',
    displayProps: displayPropsDeeplink,
    filter(v: KrefiaDeepLink) {
      return v.type === 'schuldhulp';
    },
  },
  lening: {
    title: 'Leningen',
    displayProps: displayPropsDeeplink,
    filter(v: KrefiaDeepLink) {
      return v.type === 'lening';
    },
  },
  budgetbeheer: {
    title: 'Budgetbeheer',
    displayProps: displayPropsDeeplink,
    filter(v: KrefiaDeepLink) {
      return v.type === 'budgetbeheer';
    },
  },
} as const;
