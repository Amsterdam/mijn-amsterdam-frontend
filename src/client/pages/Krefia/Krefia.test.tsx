import { render, screen } from '@testing-library/react';
import { generatePath } from 'react-router-dom';
import { MutableSnapshot } from 'recoil';

import { KrefiaThemaPagina } from './Krefia';
import KrefiaData from '../../../../mocks/fixtures/krefia.json';
import {
  forTesting,
  KrefiaSourceResponse,
} from '../../../server/services/krefia';
import { AppRoutes } from '../../../universal/config/routes';
import { AppState } from '../../../universal/types';
import { appStateAtom } from '../../hooks/useAppState';
import MockApp from '../MockApp';

const testState = {
  KREFIA: {
    content: forTesting.transformKrefiaResponse(
      KrefiaData as KrefiaSourceResponse
    ),
    status: 'OK',
  },
} as unknown as AppState;

function initializeState(snapshot: MutableSnapshot, state: AppState) {
  snapshot.set(appStateAtom, state);
}

describe('<Krefia />', () => {
  const routeEntry = generatePath(AppRoutes.KREFIA);
  const routePath = AppRoutes.KREFIA;
  function Component({ state }: { state: AppState }) {
    return (
      <MockApp
        routeEntry={routeEntry}
        routePath={routePath}
        component={KrefiaThemaPagina}
        initializeState={(snap) => initializeState(snap, state)}
      />
    );
  }

  it('Shows the page succesfully', () => {
    render(<Component state={testState} />);
    expect(screen.getByText('Kredietbank & FIBU')).toBeInTheDocument();
    expect(
      screen.getByText('Meer informatie over Kredietbank Amsterdam')
    ).toBeInTheDocument();
    expect(
      screen.getByText('Afkoopvoorstellen zijn verstuurd')
    ).toBeInTheDocument();
    expect(
      screen.getByText(
        'Kredietsom €1.689,12 met openstaand termijnbedrag €79,66'
      )
    ).toBeInTheDocument();
    expect(screen.getByText('Lopend')).toBeInTheDocument();
  });
});
