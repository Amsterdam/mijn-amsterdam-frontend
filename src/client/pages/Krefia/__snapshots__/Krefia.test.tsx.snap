// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Krefia /> > Matches the Full Page snapshot 1`] = `
<DocumentFragment>
  <main
    class="_Page_47d7c0 _OverviewPage_47d7c0 _Krefia_2f2815"
  >
    <header
      class="_PageHeading_3b337d _hasIcon_3b337d _hasBackLink_3b337d"
    >
      <span
        aria-hidden="true"
        class="_Icon_3b337d"
      >
        <svg
          aria-label="Kredietbank & FIBU"
          class="_ThemaIcon_2fc54d"
          fill="black"
          focusable="false"
          height="30px"
          viewBox="0 0 32 30"
          width="32px"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M20,21.0001001 C21.0764827,21.0001001 22,21.4542017 22,23.0001001 C22,24.5459986 20.8882453,25.0001001 20,25.0001001 L15,25.0001001 C13.6666667,25.0001001 13,25.6667668 13,27.0001001 L21,27.0001001 L21,27.0001001 C21.6203613,27.0001001 21.8165644,26.7720963 21.9648986,26.6580944 C23.4811102,25.4928132 25.7554276,23.7448914 28.7878507,21.414329 C29.663648,20.7412369 30.9192709,20.9055618 31.592363,21.7813591 C32.2654551,22.6571563 32.1011301,23.9127792 31.2253329,24.5858713 L24.0381179,30.1095902 C23.4379709,30.5708319 22.608178,30.9706998 21.3089708,30.9789733 C21.2557449,30.982159 21.1964406,30.9841201 21.1296887,30.9850735 L-2.84217094e-14,30.985775 L-2.84217094e-14,30.985775 L-2.84217094e-14,25.0001001 L3.98706055,25.0001001 L6.55301511,22.5471764 C7.71185171,21.5471651 9.16480928,21.0001001 10.6648262,21.0001001 L20,21.0001001 Z M16,1 C20.9705627,1 25,5.02943725 25,10 C25,14.9705627 20.9705627,19 16,19 C11.0294373,19 7,14.9705627 7,10 C7,5.02943725 11.0294373,1 16,1 Z M16,3 C12.1340068,3 9,6.13400675 9,10 C9,13.8659932 12.1340068,17 16,17 C19.8659932,17 23,13.8659932 23,10 C23,6.13400675 19.8659932,3 16,3 Z M16.6005362,6 C16.9151043,6 17.2332424,6.02859668 17.5549598,6.08579088 C17.8766772,6.14298509 18.1411965,6.22162595 18.3485255,6.32171582 L17.8230563,7.67292225 C17.6657722,7.59428021 17.4709574,7.53351228 17.2386059,7.49061662 C17.0062544,7.44772097 16.7756937,7.42627346 16.5469169,7.42627346 C16.3610357,7.42627346 16.1787319,7.44950826 16,7.49597855 C15.8212681,7.54244885 15.6550499,7.61036595 15.5013405,7.6997319 C15.347631,7.78909786 15.2100095,7.90169727 15.0884718,8.03753351 C14.9669342,8.17336976 14.873995,8.32707689 14.8096515,8.49865952 L17.6514745,8.49865952 L17.2761394,9.61394102 L14.5093834,9.61394102 C14.4950848,9.6782845 14.4879357,9.74083973 14.4879357,9.80160858 L14.4879357,10.1876676 C14.4879357,10.2555857 14.4915102,10.3181409 14.4986595,10.3753351 L17.0831099,10.3753351 L16.7184987,11.4798928 L14.7238606,11.4798928 C14.7739055,11.6300276 14.8525464,11.7712236 14.9597855,11.9034853 C15.0670247,12.0357469 15.1992843,12.1501336 15.3565684,12.2466488 C15.5138524,12.343164 15.6890071,12.4182303 15.8820375,12.4718499 C16.075068,12.5254694 16.2788193,12.5522788 16.4932976,12.5522788 C17.022344,12.5522788 17.4763162,12.4414667 17.8552279,12.2198391 L18.2520107,13.5817694 C18.0446817,13.7033071 17.7783752,13.8033955 17.4530831,13.8820375 C17.127791,13.9606796 16.7649707,14 16.3646113,14 C15.892759,14 15.4691708,13.9356575 15.0938338,13.8069705 C14.7184968,13.6782835 14.3914223,13.5013416 14.1126005,13.2761394 C13.8337788,13.0509372 13.5996435,12.7846307 13.4101877,12.4772118 C13.2207318,12.1697929 13.0795358,11.8373566 12.9865952,11.4798928 L12,11.4798928 L12.3646113,10.3753351 L12.847185,10.3753351 C12.8400357,10.3109916 12.8364611,10.2466491 12.8364611,10.1823056 L12.8364611,10 C12.8364611,9.87131303 12.8471849,9.74262799 12.8686327,9.61394102 L12.0321716,9.61394102 L12.4182306,8.49865952 L13.0294906,8.49865952 C13.143879,8.10544933 13.3118845,7.75514005 13.5335121,7.44772118 C13.7551396,7.14030231 14.0214462,6.87935764 14.3324397,6.66487936 C14.6434332,6.45040107 14.9901679,6.28597019 15.3726542,6.17158177 C15.7551404,6.05719335 16.1644303,6 16.6005362,6 Z"
          />
        </svg>
      </span>
      <div
        class="_HeadingInner_3b337d"
      >
        <a
          class="ams-link ams-link--standalone _BackLink_3b337d"
          href="/"
        >
          <span
            class="ams-icon ams-icon--size-5"
          >
            <svg
              aria-hidden="true"
              focusable="false"
              viewBox="0 0 32 32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m22.857 32-16-16 16-16 2.9 2.91L12.677 16l13.08 13.09z"
                fill-rule="evenodd"
              />
            </svg>
          </span>
          <span>
            Home
          </span>
        </a>
        <h2
          class="ams-heading ams-heading--level-1"
        >
          Kredietbank & FIBU
        </h2>
      </div>
    </header>
    <div
      class="_PageContent_47d7c0"
    >
      <p>
        Een online plek waar u alle informatie over uw geldzaken kunt vinden als klant van Budgetbeheer (FIBU) en/of Kredietbank Amsterdam.
      </p>
      <p>
        <a
          class="_Button_c3ab45 _Button__plain_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45"
          href="https://www.amsterdam.nl/werk-inkomen/kredietbank-amsterdam/"
          rel="external noopener noreferrer"
        >
          <svg
            aria-hidden="true"
            aria-label="chevron-rechts-icoon"
            class="_Icon_c3ab45 _Icon__left_c3ab45"
            fill="#000"
            focusable="false"
            height="14px"
            viewBox="0 0 32 32"
            width="14px"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <polygon
              points="9.75688579 32 6.85714286 29.0899743 19.9368344 16 6.85714286 2.91002571 9.75688579 0 25.7568858 16"
            />
          </svg>
          Meer informatie over Kredietbank Amsterdam
        </a>
        <br />
        <a
          class="_Button_c3ab45 _Button__plain_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45"
          href="https://www.amsterdam.nl/werk-inkomen/bijstandsuitkering/budgetbeheer"
          rel="external noopener noreferrer"
        >
          <svg
            aria-hidden="true"
            aria-label="chevron-rechts-icoon"
            class="_Icon_c3ab45 _Icon__left_c3ab45"
            fill="#000"
            focusable="false"
            height="14px"
            viewBox="0 0 32 32"
            width="14px"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <polygon
              points="9.75688579 32 6.85714286 29.0899743 19.9368344 16 6.85714286 2.91002571 9.75688579 0 25.7568858 16"
            />
          </svg>
          Meer informatie over Budgetbeheer (FIBU)
        </a>
      </p>
    </div>
    <section
      class="_SectionCollapsible_b93dc7 _SectionBorderTop_2f2815"
    >
      <h3
        class="ams-heading ams-heading--level-3 _Title_b93dc7 _TitleWithItems_b93dc7"
      >
        <button
          aria-expanded="true"
          class="_TitleToggle_b93dc7"
        >
          <svg
            aria-hidden="true"
            aria-label="chevron-rechts-icoon"
            class="_CaretIcon_b93dc7"
            focusable="false"
            height="32px"
            viewBox="0 0 32 32"
            width="32px"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <polygon
              points="9.75688579 32 6.85714286 29.0899743 19.9368344 16 6.85714286 2.91002571 9.75688579 0 25.7568858 16"
            />
          </svg>
           Schuldregeling
        </button>
      </h3>
      <div
        aria-hidden="false"
        class="_Panel_b93dc7"
        style="height: 0px;"
      >
        <div
          class="_PanelInner_b93dc7"
        >
          <table
            class="_Table_a66dcc _HulpTable_2f2815"
          >
            <thead>
              <tr
                class="_TableRow_a66dcc"
              >
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Status
                </th>
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Bekijk op Krefia
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="_TableRow_a66dcc"
              >
                <td
                  class="_DisplayPropTitle_a66dcc"
                >
                  Afkoopvoorstellen zijn verstuurd
                </td>
                <td
                  class="_DisplayProp_a66dcc"
                >
                  <span
                    class="_DisplayPropLabel_a66dcc"
                  >
                    Bekijk op Krefia:
                  </span>
                  <a
                    class="_Button_c3ab45 _Button__inline_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45 _LinkedInline_c3ab45"
                    href="http://host/srv/2442531/2"
                    rel="external noopener noreferrer"
                  >
                    Bekijk uw schuldregeling
                  </a>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
    <section
      class="_SectionCollapsible_b93dc7"
    >
      <h3
        class="ams-heading ams-heading--level-3 _Title_b93dc7 _TitleWithItems_b93dc7"
      >
        <button
          aria-expanded="false"
          class="_TitleToggle_b93dc7"
        >
          <svg
            aria-hidden="true"
            aria-label="chevron-rechts-icoon"
            class="_CaretIcon_b93dc7"
            focusable="false"
            height="32px"
            viewBox="0 0 32 32"
            width="32px"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <polygon
              points="9.75688579 32 6.85714286 29.0899743 19.9368344 16 6.85714286 2.91002571 9.75688579 0 25.7568858 16"
            />
          </svg>
           Leningen
        </button>
      </h3>
      <div
        aria-hidden="true"
        class="_Panel_b93dc7"
        style="height: 0px;"
      >
        <div
          class="_PanelInner_b93dc7"
        >
          <table
            class="_Table_a66dcc _HulpTable_2f2815"
          >
            <thead>
              <tr
                class="_TableRow_a66dcc"
              >
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Status
                </th>
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Bekijk op Krefia
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="_TableRow_a66dcc"
              >
                <td
                  class="_DisplayPropTitle_a66dcc"
                >
                  Kredietsom €1.689,12 met openstaand termijnbedrag €79,66
                </td>
                <td
                  class="_DisplayProp_a66dcc"
                >
                  <span
                    class="_DisplayPropLabel_a66dcc"
                  >
                    Bekijk op Krefia:
                  </span>
                  <a
                    class="_Button_c3ab45 _Button__inline_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45 _LinkedInline_c3ab45"
                    href="http://host/pl/2442531/1"
                    rel="external noopener noreferrer"
                  >
                    Bekijk uw lening
                  </a>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
    <section
      class="_SectionCollapsible_b93dc7"
    >
      <h3
        class="ams-heading ams-heading--level-3 _Title_b93dc7 _TitleWithItems_b93dc7"
      >
        <button
          aria-expanded="false"
          class="_TitleToggle_b93dc7"
        >
          <svg
            aria-hidden="true"
            aria-label="chevron-rechts-icoon"
            class="_CaretIcon_b93dc7"
            focusable="false"
            height="32px"
            viewBox="0 0 32 32"
            width="32px"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <polygon
              points="9.75688579 32 6.85714286 29.0899743 19.9368344 16 6.85714286 2.91002571 9.75688579 0 25.7568858 16"
            />
          </svg>
           Budgetbeheer
        </button>
      </h3>
      <div
        aria-hidden="true"
        class="_Panel_b93dc7"
        style="height: 0px;"
      >
        <div
          class="_PanelInner_b93dc7"
        >
          <table
            class="_Table_a66dcc _HulpTable_2f2815"
          >
            <thead>
              <tr
                class="_TableRow_a66dcc"
              >
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Status
                </th>
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Bekijk op Krefia
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="_TableRow_a66dcc"
              >
                <td
                  class="_DisplayPropTitle_a66dcc"
                >
                  Lopend
                </td>
                <td
                  class="_DisplayProp_a66dcc"
                >
                  <span
                    class="_DisplayPropLabel_a66dcc"
                  >
                    Bekijk op Krefia:
                  </span>
                  <a
                    class="_Button_c3ab45 _Button__inline_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45 _LinkedInline_c3ab45"
                    href="http://host/bbr/2064866/3"
                    rel="external noopener noreferrer"
                  >
                    Ga naar budgetbeheer
                  </a>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
  </main>
</DocumentFragment>
`;
